content/influxdb/cloud/upgrade/v1-to-cloud/_index.md

```diff
@@ -0,0 +1,259 @@
+
+1. [Create an InfluxDB Cloud account](#create-an-influxdb-cloud-account)
+2. [Create an All-Access authentication token](#create-an-all-access-authentication-token)
+3. [Download and install the `influx` CLI](#download-and-install-the-influx-cli)
+4. [Create DBRP mappings](#create-dbrp-mappings)
+5. [Dual write to InfluxDB 1.x and InfluxDB Cloud](#dual-write-to-influxdb-1x-and-influxdb-cloud)
+6. [Migrate time series data](#migrate-time-series-data)
+7. [Migrate continuous queries](#migrate-continuous-queries)
+8. [Collaborate with other users](#collaborate-with-other-users)
+
+## Create an InfluxDB Cloud account
+Do one of the following to create an InfluxDB Cloud account:
+
+- [Subscribe through InfluxData](/influxdb/cloud/get-started/#subscribe-through-influxdata) and
+  [start for free](/influxdb/cloud/get-started/#start-for-free).
+- [Subscribe through your cloud provider](/influxdb/cloud/get-started/#subscribe-through-a-cloud-provider).
+
+## Create an All-Access authentication token
+InfluxDB Cloud requires all requests to be authenticated with **token authentication**.
+Create an **All-Access** token in your InfluxDB Cloud user interface (UI) to use
+for the upgrade process.
+
+1. Click **Data (Load Data) > Tokens** in the left navigation bar.
+
+    {{< nav-icon "data" >}}
+2. Click **{{< icon "plus" >}} Generate**, and then select **All-Access Token**.
+3. Enter a description for the token, and then click **{{< icon "check" >}} Save**.
+
+_For more information about managing tokens and token types, see [Manage tokens](/influxdb/cloud/security/tokens/)._
+
+## Download and install the influx CLI
+Visit the [InfluxDB downloads page](https://portal.influxdata.com/downloads/)
+and download the **InfluxDB Cloud CLI** (`influx`).
+Place the `influx` binary in your system `PATH` or execute the CLI commands from
+the directory where the `influx` CLI exists.
+
+[Create a CLI connection configuration](/influxdb/cloud/reference/cli/influx/#provide-required-authentication-credentials)
+for your InfluxDB Cloud account.
+Include the following flags:
+
+- **-\-config-name**:
+  Unique name for the connection configuration.
+- **-\-host-url**:
+  [InfluxDB Cloud region URL](/influxdb/cloud/reference/regions/).
+- **-\-org**:
+  InfluxDB Cloud organization name.
+  The default organization name is the email address associated with your account.
+- **-\-token**:
+  InfluxDB Cloud **All-Access** token.
+
+```sh
+influx config create \
+  --config-name cloud \
+  --host-url https://cloud2.influxdata.com \
+  --org your.email@example.com \
+  --token mY5uP3rS3cRe7Cl0uDt0K3n \
+  --active
+```
+
+{{% note %}}
+#### Required InfluxDB Cloud credentials
+All `influx` CLI examples below assume the required InfluxDB Cloud **host**,
+**organization**, and **authentication token** credentials are provided by your
+[`influx` CLI configuration](/influxdb/cloud/reference/cli/influx/#provide-required-authentication-credentials).
+{{% /note %}}
```

Comment:
Good best practice! Perhaps we could link back to this note in context of examples? If folks land on "Create DBRP mappings," for example, from elsewhere and haven't set up their config, they may not see this note...think we've had a couple users run into this issue. I know this is a global issue and hairy one to address.

Reply:
I added a new caption shortcode that lets you put a caption under images and code blocks. I used this under each of the v2 CLI examples.